### ISSUE TITLE
[aspectj-plugin] Does not handle additional source sets when using Groovy or Scala plugins. 

### ISSUE THREAD

**Issue by emopti-jrufer** [freefair/gradle-plugins#329]
Adding additional source set causes the following error. Noticed when trying to add an additional source set for integration tests into a project.

```
Task with name 'compileFooGroovy' not found in root project

Caused by: org.gradle.api.UnknownTaskException: Task with name 'compileFooGroovy' not found in root project 'aspectj-poc'.
	at io.freefair.gradle.plugins.aspectj.AspectJPostCompileWeavingPlugin.lambda$configureSourceSet$4(AspectJPostCompileWeavingPlugin.java:68)
	at io.freefair.gradle.plugins.aspectj.AspectJPostCompileWeavingPlugin.configureSourceSet(AspectJPostCompileWeavingPlugin.java:67)
	at build_824we5p5thu23fphjxhfpnswe$_run_closure2.doCall(/projects/aspectj-poc/build.gradle:13)
```

```gradle
plugins {
    id 'java'
    id 'groovy'
    id 'io.freefair.aspectj.post-compile-weaving' version '5.3.3.3'
}

repositories {
    mavenCentral()
}

sourceSets {
    foo {
    }
}

```

**Comment by larsgrefer** [comment]
Thanks for pointing this out

**Comment by emopti-jrufer** [comment]
@larsgrefer thank you for getting on that so fast. I also had a fix and was about to create a PR for it. I merged my changes with yours with tests in #330.
